security-release-process.md

```diff
@@ -225,6 +225,10 @@ These steps should be completed within the 1-7 days of Disclosure.
 
 If the CVSS score is under 4.0 ([a low severity score](https://www.first.org/cvss/specification-document#i5)) or the assessed risk is low the Fix Team can decide to slow the release process down in the face of holidays, developer bandwidth, etc. These decisions must be discussed on the security@kubernetes.io mailing list.
 
+If the CVSS score is under 7.0 (a medium severity score), the Fix Lead may choose to carry out the fix semi-publicly. This means that PRs are made directly in the public kubernetes/kubernetes repo, while restricting discussion of the security aspects to private channels. The fix lead will make the determination whether there would be user harm in handling the fix publicly that outweighs the benefits of open engagement with the community.
+
+If the vulnerability requires [User Interaction](https://www.first.org/cvss/user-guide#5-4-User-Interaction), especially in client components like kubectl, the Fix Lead may choose to disclose the vulnerability before a fix is developed if they determine that users would be better off being warned against a specific interaction.
```

Comment:
Would this only apply to Medium and below, or to any vulns?
I have no opposition for Medium and below; in general, I'd be concerned about sharing until there is a fix available.

Reply:
I didn't intend to restrict it to Medium and below. I could conceive of some Critical vuln that we would be better off spreading as quickly and as widely as we can, even if the fix wouldn't be out for a week.

Silly example to try to highlight what I'm thinking: If `kubectl --v=9` caused your credentials to be sent to a random place in plaintext, we would want to tell as many people as quickly as possible "Don't use `--v=9` w/ your kubectl calls." (that contrived example gets me a [9.3 CVSS](https://www.first.org/cvss/calculator/3.0#CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:C/C:H/I:H/A:N) score)

I couldn't think of a reason why withholding that info until we had a patch would benefit anyone. With some "client-side" things, I worry about their use in server applications (e.g. using kubectl or the client libs in an Operator). But, when rolling out a full mitigation is no harder than rolling out a patch, I don't think we gain anything by waiting for the patch to be available.

I don't think this flexibility applies to vulnerabilities that aren't easy to mitigate. E.g. if the vuln was "any kubectl call using client certs exposes keys," we can't just tell people "stop using your authN method."

Reply:
Got it - it just looked like it followed as another rating category. Suggest you maybe rephrase to, "No matter the CVSS score, if the vulnerability..."

Reply:
Okay. Clarified.